### PR TITLE
Updated translation teams matching

### PR DESCRIPTION
**Pull request by biodranik**
@rtsisyk please add missing teams.

### CODE REVIEW

**Comment by vng** [comment]
Пишет "This CODEOWNERS file contains errors"

**Comment by biodranik** [comment]
Да, надо добавить тимы, там написано, какие.

**Comment by rtsisyk** [comment]
GitHub complains about empty teams... I am re-adding myself to all teams without users.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -38,12 +43,13 @@ android/res/values-zh-rTW/ @organicmaps/translations-zh-Hant
...
 iphone/Maps/LocalizedStrings/be.lproj/ @organicmaps/translations-be
 iphone/Maps/LocalizedStrings/bg.lproj/ @organicmaps/translations-bg
+iphone/Maps/LocalizedStrings/ca.lproj/ @organicmaps/translations-ca
 iphone/Maps/LocalizedStrings/cs.lproj/ @organicmaps/translations-cs
 iphone/Maps/LocalizedStrings/da.lproj/ @organicmaps/translations-da
 iphone/Maps/LocalizedStrings/de.lproj/ @organicmaps/translations-de
 iphone/Maps/LocalizedStrings/el.lproj/ @organicmaps/translations-el
 iphone/Maps/LocalizedStrings/es.lproj/ @organicmaps/translations-es
-iphone/Maps/LocalizedStrings/es-MX.lproj/ @organicmaps/translations-es
+iphone/Maps/LocalizedStrings/es-MX.lproj/ @organicmaps/translations-es-MX
```
Comment (rtsisyk) [inline .github/CODEOWNERS:52]: Do we have enough people to have separate teams for es and es-MX? I have doubts about that.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -72,13 +79,16 @@ iphone/Maps/LocalizedStrings/zh-Hans.lproj/ @organicmaps/translations-zh-Hans
...
 android/src/fdroid/play/listings/ca/ @organicmaps/translations-ca
 android/src/fdroid/play/listings/cs-CZ/ @organicmaps/translations-cs
+android/src/fdroid/play/listings/de-DE/ @organicmaps/translations-de
 android/src/fdroid/play/listings/eu-ES/ @organicmaps/translations-eu
 android/src/fdroid/play/listings/fr-FR/ @organicmaps/translations-fr
 android/src/fdroid/play/listings/it-IT/ @organicmaps/translations-it
+android/src/fdroid/play/listings/id/ @organicmaps/translations-id
 android/src/fdroid/play/listings/mr-IN/ @organicmaps/translations-mr
-android/src/fdroid/play/listings/pt-BR/ @organicmaps/translations-pt
+android/src/fdroid/play/listings/pt-BR/ @organicmaps/translations-pt-BR
```
Comment (rtsisyk) [inline .github/CODEOWNERS:91]: The same questions here. Do we have enough people to have separate teams?

**Review by rtsisyk — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (biodranik) [reply .github/CODEOWNERS:52]: We WILL have them sooner or later. It is easier to exactly match our existing languages, and then add contributors when we find them (for example, when someone contributes changes to these languages, he/she can be asked and added into that team).

File: .github/CODEOWNERS (hunk 2 above)
Comment (biodranik) [reply .github/CODEOWNERS:91]: ditto ^^^

File: .github/CODEOWNERS (hunk 1 above)
Comment (rtsisyk) [reply .github/CODEOWNERS:52]: Are `es` and `es-MX` really require two completely separate groups of people to review? I have doubts.

Comment (rtsisyk) [reply .github/CODEOWNERS:52]: I've fixed master. Are you sure that we actually need to split `es` and `es-MX`? I am not a big expert in Español, but I thought that Mexican variation is not significantly different. It is not like Simplified vs. Traditional Chinese.

Comment (biodranik) [reply .github/CODEOWNERS:52]: There are many different words in EU and Latin America.

It is the same as GB and US case.

Comment (rtsisyk) [reply .github/CODEOWNERS:52]: Let's get to the point when we have 2+ folks from both side and when split the team. Currently this is a premature optimization nobody asked us to do.

Comment (biodranik) [reply .github/CODEOWNERS:52]: It's inevitable and only a question of time. Now it will take 5 min of your time. Later it will take 25 min.

**Pull request merged**